src/components/VBtn/VBtn.vue

```diff
@@ -125,6 +126,10 @@
       tag === 'button' && (data.attrs.type = this.type)
       this.loading && children.push(this.genLoader())
 
+      if (typeof this.value !== 'undefined') {
+        data.attrs.value = (typeof this.value === 'object') ? JSON.stringify(this.value) : this.value
```

Comment:
@johnleider we decided in discord that the serialisation is a bad idea and can just be
```js
data.attrs.value = this.value
```

Reply:
updated, ty